app/controllers/teams_controller.rb

```diff
@@ -76,11 +76,32 @@ def delete_membership
     redirect_to @team
   end
 
+  def assign_membership_by_email
+    user = User.find_by_email params[:email]
+    if user
+      assign_membership_to_user user
+    else
+      flash[:error] = I18n.t('teams.invite_user_to_team.no_user_for_mail')
+    end
+    redirect_to @team
+  end
+
   def current_ability
     @current_ability ||= Ability.new(current_user, params[:team_member])
   end
 
   private
+    def assign_membership_to_user (new_member)
+      team_members = @team.members
+      unless team_members.include? new_member
+        team_members << new_member
+        # todo: send mail
```

Comment:
How is the status here? Do you implement custom emails or do you share code with @MerlindlH's team? Do they already have implemented custom emails? Do you plan to finish this feature in the current sprint or is this going to be a new issue?

Reply:
As far as I know custom emails hasn't been implemented by @MerlindlH's team yet. Though we could implement it by ourselves this would go beyond the scope of this issue (in my opinion). So I would suggest we consider it as follow up and reestimate on Wednesday.

Reply:
The acceptance criterias say: "The invited person receives a notification email". In my opinion we can consider it as follow up and reestimate. I would suggest to talk to PO today and ask for clarification of the acceptance criterias.

Reply:
Yeah you're right regarding the criterias but I'm pretty sure we talked about the pre conditions as "logic for sending custom emails is already implemented" when we estimated this issue... But yes, let's talk with @svitlanakalancha about this.

Reply:
@lheilmann We talked with @svitlanakalancha about the email notifications before implementing the issue – sorry for not including you in the discussion in the first place! I added you to the corresponding Slack channel now. Here is a screenshot of the discussion:

<img width="622" alt="email_notifications" src="https://user-images.githubusercontent.com/807665/33833425-e1da2f5e-de7f-11e7-8315-93cf95ecaa49.png">

Reply:
Okay. Thanks for the screenshot! :+1: If you could find a solution by talking to @svitlanakalancha, that's perfect. For the next time: Please notify the corresponding reviewers or (better) give updates in the issue. :+1: